LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 santey
+Copyright (c) 2017 GMV Syncromatics Engineering
```

Comment:
This is a special case where we need to preserve the original license. Can you change this back to the original copyright?